shared/src/metabase/shared/util/time.cljc

```diff
@@ -14,6 +14,7 @@
 (shared.ns/import-fn internal/same-day?)
 (shared.ns/import-fn internal/same-month?)
 (shared.ns/import-fn internal/same-year?)
+(shared.ns/import-fn internal/day-diff)
```

Comment:
This code predates the more advanced `import-fns` macro. No need to change it in this PR, but if you have to make changes you can throw that in as well.

Reply:
Will fix in a different PR.